src/angular/src/swiper.component.ts

```diff
@@ -459,7 +459,8 @@ export class SwiperComponent implements OnInit {
   ) {}
 
   private _setElement(el: ElementRef, ref: any, update: string, key = 'el') {
-    if (ref && el && el.nativeElement) {
+    if (!ref && !el) return;
+    if (el.nativeElement) {
```

Comment:
I have error in this line 
`core.js:6486 ERROR TypeError: Cannot read properties of undefined (reading 'nativeElement')`
can we convert this line to 
`if(el?.nativeElement) {`